atom/browser/web_contents_preferences.cc

```diff
@@ -135,6 +138,19 @@ void WebContentsPreferences::AppendExtraCommandLineSwitches(
       LOG(ERROR) << "preload url must be file:// protocol.";
   }
 
+  v8::Isolate* isolate = v8::Isolate::GetCurrent();
+  mate::Handle<atom::api::WebContents> api_web_contents =
+    atom::api::WebContents::CreateFrom(isolate, web_contents);
+  auto session = atom::api::Session::CreateFrom(
+    isolate, api_web_contents.get()->GetBrowserContext());
+  for (auto preloadPath : session->GetPreloads()) {
+    if (base::FilePath(preloadPath).IsAbsolute())
```

Comment:
Would it be better to do this check in `AddPreload()`?

Reply:
For consistency with existing preload logic I think it's better to leave this here